docs/data/material/components/icons/TwoToneIcons.js

```diff
@@ -12,8 +12,8 @@ export default function TwoToneIcons() {
 
   return (
     <Icon
-      sx={{ ...(isDarkMode && { filter: 'invert(1)' }) }}
       baseClassName="material-icons-two-tone"
+      style={isDarkMode ? { filter: 'invert(1)' } : undefined}
```

Comment:
I don't think we should change this. We should create a fix from Pigment CSS first and then apply it to this PR. It's okay to have this PR open and label it as `on hold` because it is blocked by another issue.

Reply:
Agreed, let's wait.

Reply:
I reverted this change.

Reply:
THB, the fix will take some time. It's not just about the `sx` transform here but also about the support for transformed `sx` prop to be passed to the `Icon` that Marija is working on.
Even right now, one particular transform is supported, that is, `sx={isDarkMode ? {} : undefined}`. This would work as far as transformation is concerned. But we'll still need to support the `sx` prop in underlying component.
If there is an alternate, we should do that and not wait for a proper fix.

Reply:
Note: moving from `sx` to `style` changes the specificity.

Reply:
True. But it's a component in our docs, not a generic component or library component. So it should be fine since it doesn't affect end-users.